docs/shell-tips.md

```diff
@@ -158,16 +191,146 @@ and hit <kbd>ENTER</kbd>/<kbd>RETURN</kbd> at the end.
 
 If you think about it, what is happening here is actually another case of
 character escaping. The newline character (the one that appears when you hit
-<kbd>ENTER</kbd>) usually means "read this command". However, here we want to literally
-have the newline character, and thus the `\<newline>`.
+<kbd>ENTER</kbd>) usually means "read this command". However, here we want to
+literally have the newline character, and thus the `\<newline>`.
```

Comment:
That definition doesn't seem to be accurate (you don't need to use quotes to escape characters).
Here you have some examples:

```
$ echo -e $PATH
$ echo -e \$PATH
$ cd ~/my\ directory\ with\ spaces
```